packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -230,14 +229,14 @@ const performRender = async (req, res, next) => {
         return next(e)
     }
 
-    res.__performanceTimer.mark(PERFORMANCE_MARKS.renderToString, 'end')
     // Step 5 - Determine what is going to happen, redirect, or send html with
     // the correct status code.
     const {html, routerContext, error} = renderResult
     const redirectUrl = routerContext.url
     const status = (error && error.status) || res.statusCode
-
+    res.__performanceTimer.mark(PERFORMANCE_MARKS.renderToString, 'end')
```

Comment:
Out of curiosity what was the reason for moving the `renderToString` performance marker here?